lua/jj/status.lua

```diff
@@ -0,0 +1,101 @@
+local from_entry = require("telescope.from_entry")
+local putils = require("telescope.previewers.utils")
+local diff_files = function(opts)
+    return require("telescope.previewers").new_buffer_previewer({
+        title = "JJ File Diff Preview",
+        get_buffer_by_name = function(_, entry)
+            return entry.value
+        end,
+
+        define_preview = function(self, entry)
+            if entry.status and (entry.status == "??" or entry.status == "A ") then
+                local p = from_entry.path(entry, true, false)
+                if p == nil or p == "" then
+                    return
+                end
+                conf.buffer_previewer_maker(p, self.state.bufnr, {
+                    bufname = self.state.bufname,
+                    winid = self.state.winid,
+                    preview = opts.preview,
+                    file_encoding = opts.file_encoding,
+                })
+            else
+                local cmd = utils.get_os_command_output(
+                    { "jj", "diff", "--no-pager", entry.value },
+                    opts.cwd
+                )
+                putils.job_maker(cmd, self.state.bufnr, {
+                    value = entry.value,
+                    bufname = self.state.bufname,
+                    cwd = opts.cwd,
+                    callback = function(bufnr)
+                        if vim.api.nvim_buf_is_valid(bufnr) then
+                            putils.highlighter(bufnr, "diff", opts)
+                        end
+                    end,
+                })
+            end
+        end,
+    })
+end
+
+local jj_show = function(opts)
+    local cmd = { "jj", "show", "--summary", "--no-pager", "-T", '""' }
```

Comment:
Maybe `diff` is better?

![image](https://github.com/zschreur/telescope-jj.nvim/assets/2672503/55665a99-5281-42fb-8b22-68ce8660cbb0)